File: app/src/main/java/com/neome/java/api/home/aside/msg/MsgGroupPatch.java
// neome.ai API - do not change
//

package com.neome.java.api.home.aside.msg;

import com.neome.java.api.core.base.msg.MsgVersion;
import com.neome.java.api.home.base.Types.EnumGroupPatchPropName;
import com.neome.java.api.home.base.dto.DtoGroupSettings;
import com.neome.java.api.meta.base.Types.GroupId;
import com.neome.java.api.meta.base.Types.MediaIdAvatar;

import org.jetbrains.annotations.Nullable;

import java.util.Set;

@SuppressWarnings("unused")
public class MsgGroupPatch extends MsgVersion
{
  @Nullable
  public String about;

  public GroupId groupId;

  @Nullable
  public MediaIdAvatar mediaIdAvatar;

  @Nullable
  public String name;

  public Set<EnumGroupPatchPropName> patchPropNameSet;

  @Nullable
  public DtoGroupSettings settings;
}

File: app/src/main/java/com/neome/api/home/aside/msg/MsgGroupPatch.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.home.aside.msg

import com.neome.api.home.base.dto.DtoGroupSettings
import com.neome.api.home.base.Types.EnumGroupPatchPropName
import com.neome.api.meta.base.Types.GroupId
import com.neome.api.meta.base.Types.MediaIdAvatar
import com.neome.api.core.base.msg.MsgVersion

interface MsgGroupPatch : MsgVersion
{
  val about: String?
  val groupId: GroupId
  val mediaIdAvatar: MediaIdAvatar?
  val name: String?
  val patchPropNameSet: Set<EnumGroupPatchPropName>
  val settings: DtoGroupSettings?
}